tests/js/spec/views/team/teamMembers.spec.jsx

```diff
@@ -44,153 +43,98 @@ describe('TeamMembers', function () {
     });
   });
 
-  it('renders', async function () {
-    const wrapper = mountWithTheme(
-      <TeamMembers
-        params={{orgId: organization.slug, teamId: team.slug}}
-        organization={organization}
-      />,
-      routerContext
-    );
-    await tick();
-    wrapper.update();
-  });
-
   it('can add member to team with open membership', async function () {
     const org = TestStubs.Organization({access: [], openMembership: true});
-    const wrapper = mountWithTheme(
-      <TeamMembers params={{orgId: org.slug, teamId: team.slug}} organization={org} />,
-      routerContext
+    render(
+      <TeamMembers params={{orgId: org.slug, teamId: team.slug}} organization={org} />
     );
 
-    await tick();
-    wrapper.update();
-
-    wrapper.find('DropdownButton[data-test-id="add-member"]').simulate('click');
-    wrapper.find('StyledUserListElement').first().simulate('click');
-
-    await tick();
-    wrapper.update();
+    userEvent.click(await screen.findByTestId('add-member'));
```

Comment:
this is existing code but I feel like we overuse test id and under use aria attributes (esp w/ buttons where we should probably use aria label).